### PR TITLE
test

### PR DESCRIPTION
**Pull request by seungho-jo**
첫 pull request

### CODE REVIEW

**Review by joseungho123 — COMMENTED** [review]
pass

**Review by joseungho123 — COMMENTED** [review]
perfect

File: src/main/webapp/test2.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+</script>
+</head>
+<body>
+branch를 이용한 pull request
+</body>
+<script type="text/javascript">
+
+</script>
+</html>
```
Comment (seungho-jo) [inline src/main/webapp/test2.html:16]: perfect

**Review by seungho-jo — COMMENTED** [review]
test

**Review by joseungho123 — APPROVED** [review]
tes

**Pull request merged**